pandas-stubs/core/reshape/pivot.pyi

```diff
@@ -4,20 +4,26 @@ from typing import (
     Literal,
     Sequence,
     TypeVar,
+    Union,
+    overload,
 )
 
 import numpy as np
+import pandas as pd
 from pandas.core.frame import DataFrame
 from pandas.core.groupby.grouper import Grouper
 from pandas.core.series import Series
+from typing_extensions import TypeAlias
 
 from pandas._typing import (
+    AnyArrayLike,
     HashableT,
     IndexLabel,
     Scalar,
-    npt,
 )
 
+_ExtendedAnyArrayLike: TypeAlias = Union[AnyArrayLike, pd.Categorical]
```

Comment:
I think you could pass any `ExtensionArray`, not just `pd.Categorical`, so maybe this should be:
```python
_ExtendedAnyArrayLike: TypeAlias = Union[AnyArrayLike, ArrayLike]
```
since `ArrayLike` includes `ExtensionArray`

Reply:
Fixed.